test/README.md

```diff
@@ -1,93 +1,62 @@
 # Testing Strategy
 
-## Overview
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Unit Testing](#unit-testing)
+- [End to End Testing (System Tests)](#end-to-end-testing-system-tests)
+- [Quick End to End Testing](#quick-end-to-end-testing)
+
+## Introduction
 
 The purpose of this document is to describe how the application is being tested to assure quality and coverage.
 
 The application testing strategy depends on two main types of testing to achieve the objective:
 
 - Unit testing
 - End to end (e2e) testing
```

Comment:
This is especially confusing, you've called it e2e which is that the tests you refer to as quick e2e tests are called in the code